apps/admin-gui/src/app/shared/perun-nav/perun-nav.component.ts

```diff
@@ -49,6 +52,13 @@ export class PerunNavComponent implements OnInit, AfterViewInit {
     this.isDevel = this.storeService.get('isDevel');
     this.logo = this.sanitizer.bypassSecurityTrustHtml(this.store.get('logo'));
     this.logoutEnabled = this.storeService.get('log_out_enabled');
+    this.utilsService.getAppsConfig().subscribe(config => {
+      this.profileUrl = !!config.brands[0].newApps.profile ? config.brands[0].newApps.profile : null;
+    });
+  }
+
+  redirectToProfile() {
+    window.location.href = this.profileUrl;
```

Comment:
I think we should open other applications in a new tab, what do you think?
```suggestion
    window.open(this.profileUrl, '_blank');
```